### DISCUSSION TITLE
How to get the PCB?

### DISCUSSION THREAD

**Discussion by pietervanuhm** [TillFleisch/ESPHome-Philips-Smart-Coffee#73] [Q&A] [answered]
Hi,

I have zero experience with PCB boards, how do i order/make this? Prefer to not cut the cable.
Can somebody point me in the good direction?

**Comment by TillFleisch** [comment, marked as answer]
Please take a look at #58

**Reply by pietervanuhm** [reply]
ah thanks, didn't look in the discussions!

**Reply by Marquo1** [reply]
Hi Pieter, I have a few printed boards, transistors and 2 spare picoflex connectors if you want them (I'm also in Netherlands). 
You'll only need a d1 and a picoflex cable.

This is what mine looks like:
![20240721_213846](https://github.com/user-attachments/assets/fbcad775-d960-4f2c-9e8b-7726b9bef432)

This is the design, if you choose to make it yourself:
https://oshwlab.com/marquo1/coffeepcb

[Cost breakdown](https://github.com/TillFleisch/ESPHome-Philips-Smart-Coffee/discussions/58#discussioncomment-9691059)

**Reply by knschuckmann** [reply]
The link to the pcb board does not work anymore is there an alternative link you guys can share

**Reply by Marquo1** [reply]
Can you try it while logged in to oshwlab? Somehow it's not showing up to me as well, but once I log in it's there

**Reply by knschuckmann** [reply]
Apparently it still does not work even though I am signed in. ChatGPT said that it might be not public, but it is also written on the site as you can see below 

<img width="2532" height="1170" alt="IMG_2332" src="https://github.com/user-attachments/assets/d365b6b7-16d1-4592-8a08-8b018017bdb6" />

**Comment by aneisch** [comment]
@pietervanuhm and any that come later I have 3x https://oshwlab.com/tradstaat/coffeepcb populated with picoflex + transistor (not pictured) + D1 Mini + 8 wire cable available at cost ($10.50) + shipping in the USA. Email is in my profile if you want one. 
![PXL_20240720_195453644 MP](https://github.com/user-attachments/assets/aa1721aa-b653-4988-9fe2-78bf0dfe74d7)

**Reply by pietervanuhm** [reply]
Hi @aneisch,

If you still have this complete board and this is plug and play? i'm interested. How much will it cost if you sent it to the netherlands?

**Reply by knschuckmann** [reply]
The link to the pcb does not work anymore

**Comment by SchmDan** [comment]


**Reply by paolomarchiori** [reply]
I'm interested in one PCB.
